imports/plugins/core/orders/server/no-meteor/register.js

```diff
@@ -134,21 +134,11 @@ export default async function register(app) {
       status: "completed",
       workflow: "coreOrderWorkflow",
       audience: ["dashboard/orders"]
-    }, { // Standard Order Fulfillment with shipping
-      template: "OrderSummary",
-      label: "Summary",
-      workflow: "coreOrderShipmentWorkflow",
-      audience: ["dashboard/orders"]
     }, {
       template: "OrderInvoice",
       label: "Invoice",
       workflow: "coreOrderShipmentWorkflow",
       audience: ["dashboard/orders"]
```

Comment:
It seems like `OrderInvoice` and some of the others here were also deleted. Maybe most if not all of this can go?